test/functional/apps/dashboard/group2/dashboard_filter_bar.ts

```diff
@@ -140,6 +144,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.dashboard.gotoDashboardLandingPage();
         await PageObjects.dashboard.loadSavedDashboard('with filters');
         await PageObjects.header.waitUntilLoadingHasFinished();
+        await elasticChart.setNewChartUiDebugFlag(true);
+        await queryBar.submitQuery();
```

Comment:
Does the debug flag not decorate the pie slices until after the query has been submitted?